up & going/ch3.md

```diff
@@ -39,19 +39,19 @@ The claims I make regarding delegation versus inheritance come not from a dislik
 
 But the case I make regarding prototypes and delegation is a much more involved one than what I will indulge here. If you're ready to reconsider everything you think you know about JavaScript "classes" and "inheritance," I offer you the chance to "take the red pill" (*Matrix* 1999) and check out Chapters 4-6 of the *this & Object Prototypes* title of this series.
 
-## Types & Grammar
+## Tipos y Gramática
 
-The third title in this series primarily focuses on tackling yet another highly controversial topic: type coercion. Perhaps no topic causes more frustration with JS developers than when you talk about the confusions surrounding implicit coercion.
+El tercer título de esta serie primeramente se enfoca en abordar otro tema muy controversial: la coerción de tipos. Quizás ningún tópico causa mayor frustración en los desarrolladores de JS que cuando se habla acerca de las confusiones alrededor de la coerción implícita.
 
-By far, the conventional wisdom is that implicit coercion is a "bad part" of the language and should be avoided at all costs. In fact, some have gone so far as to call it a "flaw" in the design of the language. Indeed, there are tools whose entire job is to do nothing but scan your code and complain if you're doing anything even remotely like coercion.
+En gran medida, la sabiduría convencional es que la coerción implícita es una "mala parte" del lenguaje y debería evitarse a toda costa. De hecho, algunos han ido tan lejos en llamarlo una "falla" en el diseño del lenguaje. En realidad hay herramientas cuyo su único trabajo es no hacer nada más que escanear su código y quejarse si usted está haciendo algo remotamente parecido a la coerción.
 
-But is coercion really so confusing, so bad, so treacherous, that your code is doomed from the start if you use it?
+Pero es realmente la coerción tan confuso, tan malo, tan traicionero, que su código está condenado desde el momento que usted lo usa?
```

Comment:
Hi Darwing, we can use 'confusa', 'mala', 'traicionera'... Switch adjectives to these forms due you're using 'la'

Reply:
Hi @mballeng91, it is done, thanks!